myapp/tests.py

```diff
@@ -1,3 +1,3 @@
 from django.test import TestCase
-
```

Comment:
we should add tests in general. this is a good practice for developers - we will talk about this in our next meeting